lib/OpenQA/Worker/Common.pm

```diff
@@ -118,6 +118,7 @@ sub api_init {
     else {
         $url = Mojo::URL->new($host);
     }
+    $openqa_host = $url->host;
     # Relative paths are appended to the existing one
```

Comment:
This is only used when locks are used, right? Because it basically makes it impossible to run openqa on morbo port.

What speaks against OPENQA_URL?